### DISCUSSION TITLE
为MAA添加了Scoop的安装方式

### DISCUSSION THREAD

**Discussion by Darkatse** [MaaAssistantArknights/MaaAssistantArknights#954] [Ideas]
为 MAA 添加了 Win 下包管理器 Scoop 的安装方式
Scoop是什么：[*Scoop 简介 —— Windows 上体验最好的「包管理器」*](https://sspai.com/post/52496)
用Scoop可以方便的管理很多绿色软件的安装、更新和建立快捷方式。因为我用的比较多所以手动写了个MAA的应用清单加了进去~~主要还是方便我自己用~~

使用方法：
在[安装好Scoop](https://zhuanlan.zhihu.com/p/489898732)后，先添加软件仓库
```
scoop bucket add Darkatse https://github.com/Darkatse/Scoop-Darkatse.git
```
然后更新 Scoop 列表
```
scoop update
```
接下来直接安装就可以了
```
scoop install MaaAssistantArknights
```
之后的更新只需要执行
```
scoop update *
```
即可自动更新，软件的设置和map也会保留。

注意：Scoop默认安装路径在C:\Users\ 用户名 \scoop\apps，如果用户名为中文会导致MAA无法启动，这时候可以用管理员权限全局安装，即`scoop install MaaAssistantArknights -g`。这样会安装在C:\ProgramData\scoop\apps，避免了中文路径的问题。

**Comment by 134820134820** [comment]
但看了一下它安装全会安装在C盘，不会储存不够吗，多开又怎么办呢

**Reply by Darkatse** [reply]
安装Scoop时候可以通过[指定环境变量确定默认安装位置](https://scoop-docs.vercel.app/docs/getting-started/Quick-Start.html#install-scoop-to-a-custom-directory-by-changing-scoop)
```
$env:SCOOP='D:\Scoop'
[environment]::setEnvironmentVariable('SCOOP',$env:SCOOP,'User')
# run the installer
```
对于全局安装(Global)方式也是一样
```
$env:SCOOP_GLOBAL='D:\Scoop_Global'
[Environment]::SetEnvironmentVariable('SCOOP_GLOBAL', $env:SCOOP_GLOBAL, 'Machine')
scoop install -g <app>
```

至于多开，这个好像确实不行．MAA可以打开多个，但是对应的配置应该都是同一个．
